docs/data-sources/federation_federated_blocks.md

```diff
@@ -0,0 +1,79 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_federation_federated_blocks Data Source - terraform-provider-bloxone"
+subcategory: ""
```

Comment:
subcategory should be "IPAM Federation". you might have to modify the template https://github.com/infobloxopen/terraform-provider-bloxone/blob/master/templates/resources.md.tmpl and https://github.com/infobloxopen/terraform-provider-bloxone/blob/master/templates/data-sources.md.tmpl

Reply:
done